tests/utils.go

```diff
@@ -1133,6 +1133,42 @@ func RunKubectlCommand(args ...string) error {
 	return nil
 }
 
+func SkipIfNoOc() {
+	path, err := exec.LookPath("oc")
+	if err != nil {
+		Skip("Skip test that requires oc binary")
+	}
+}
+
+func RunOcCommand(args ...string) (string, error) {
```

Comment:
That function looks now great. You will have to also add a commandline flag, so that you can do similar checks for oc, like done at https://github.com/gszasz/kubevirt/blob/0dc18ddc43a28ba29d2dee24949dba431a683602/tests/utils.go#L1108 for kubectl.

Finally, use that parameter to point our tests to the oc tool which we extract from the containerized cluster.